### PR TITLE
Further clean up of sunpy projects

### PR DESCRIPTION
**Pull request by nabobalis**
I missed a few things.

### CODE REVIEW

File: _projects/2019/sunpy/helioviewer.md [hunk 1]
```diff
@@ -13,6 +12,7 @@ issues:
 mentors:
  - wafels
  - Cadair
+ - Potentially the API creator/maintainer
```
Comment (bsipocz) [inline _projects/2019/sunpy/helioviewer.md]: this won't work on the website (but then I cannot have a look as it's down for me)

File: _projects/2019/sunpy/helioviewer.md [hunk 2]
```diff
@@ -24,25 +24,20 @@ collaborating_projects:
...
-To facilitate this, we will need to remove the current code from SunPy and move it to the new affiliated package.
-#### Secondary goals
-
-* Start on a GUI-like interface for the new package, if time permits.
+What we want to do is to create a new Python package that will lightly wrap every aspect of the HelioViewer API allowing anyone to create their own helioviewer.org like GUI in Python (this could be part of the project but will depend on the progress). To facilitate this, we will need to remove the current code from SunPy and move it to the new affiliated package.
 
 #### Milestones
 
-##### Coding starts
+##### Coding starts (time table might be too slow)
```
Comment (bsipocz) [inline _projects/2019/sunpy/helioviewer.md]: I'm not sure about this note. What does it suggest for students browsing the project ideas?

File: _projects/2019/sunpy/helioviewer.md (hunk 1 above)
Comment (nabobalis) [reply _projects/2019/sunpy/helioviewer.md]: Yes it does not show up. I will remove this.

File: _projects/2019/sunpy/helioviewer.md (hunk 2 above)
Comment (nabobalis) [reply _projects/2019/sunpy/helioviewer.md]: I'll remove it too.

**Pull request merged**